templates/base.html

```diff
@@ -2,53 +2,51 @@
 
 <!doctype html>
 <html lang="en">
-
-	<head>
-
-		<!-- Required meta tags -->
-		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-		<!-- Custom CSS -->
-		<link rel='stylesheet' type='text/css' href={% static 'css/style.css' %}>
-
-		<!-- Bootstrap CSS -->
-		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-
-		<title> {% block title %}ML Playground{% endblock %}</title>
-  
-		<!-- Bootstrap 5 -->
-		<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
-					 integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
-					 crossorigin="anonymous"
-		>
-		</script>
-
-		<!-- Altair -->
-		<script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
-		<script src="https://cdn.jsdelivr.net/npm/vega-lite@4"></script>
-		<script src="https://cdn.jsdelivr.net/npm/vega-embed@6"></script>
-
-		<!-- Extra per-template head entries -->
-		{% block extra_head %}
-
-		{% endblock %}
-
-	</head>
-
-	<body>
-
-		<!-- Navigation -->
-		{% include "menu.html" %}
-
-		<!-- Page Content -->
-		<div class='container base'>
-			{% block content %}
-			content: Oops. You shouldn't be seeing this.
-			{% endblock %}
-		</div>
-
-
-	</body>
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Custom CSS -->
+    <link rel='stylesheet' type='text/css' href={% static 'css/style.css' %}>
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" 
+      rel="stylesheet" 
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+      crossorigin="anonymous"
+    >
+
+    <!-- Bootstrap 5 -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    >
+    </script>
+
+    <!-- Altair -->
+    <script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@4"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-embed@6"></script>
```

Comment:
Are all those three libraries needed in the base? Should we just add them only on the templates that need them?

Reply:
We can indeed add them only on the templates that need them. Part of the next refactoring PR once the various views will be functional and the import of libraries done in the proper html. Can comment for now if it slows down the pages.